ansible/roles/ml-data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
```

Comment:
Will this module re-clone or update the branch / tag on a second run? Since I dont see any cleanup of the git repo at he end of the task.

Reply:
done